doc/dev/crimson/crimson.rst

```diff
@@ -301,16 +298,16 @@ To use fio to test ``crimson-store-nbd``,
         --type transaction_manager \
         --uds-path ${unix_socket} &
 
-   in which,
+   Below are descriptions of these command line arguments:
 
    ``--smp``
-     how many CPU cores are used
+     The number of CPU cores to use (Symmetric MultiProcessor)
 
    ``--mkfs``
-     initialize the device first
+     Initialize the device first
```

Comment:
s/first/first./

(Add a period (if we're making sentences here, which looks likely).)

Reply:
Added, though the one above is a non-sentence without a period.  With some modern constructs application of the rules of capitalization and punctuation isn't always clear.